docs/debugging-testing/debug/debugging_mbed_os_apps.md

```diff
@@ -6,7 +6,9 @@ The simplest way to debug your code is to augment your code with [printf](../tut
 
 ## Debugging from an IDE
 
-Keil uVision natively supports debugging Mbed OS applications. To set up uVision, read [Debugging with Keil uVision](../debug-test/keil-uvision.html).
+Mbed Studio and Keil uVision natively support debugging Mbed OS applications:
+* To set up Mbed Studio, read [Debugging with Mbed Studio](../debug-test/mbed-studio.html).
```

Comment:
Will the auto-generated HTML have this name?

Reply:
I don't think we need a new page that serves only as a link to another page. You can just link from here directly to the Mbed Studio site.

Reply:
Updated, please have a look